docs/server/source/http-client-server-api.rst

```diff
@@ -137,6 +137,10 @@ Transactions
        <https://docs.bigchaindb.com/projects/py-driver/en/latest/index.html>`_
        to build a valid transaction.
 
+   A generalization of the parameter follows:
+
+   :param mode: (Optional) Set the broadcast method.
```

Comment:
It wasn't clear to me what these two lines meant until later. At first, I thought you'd somehow forgotten to include the `?mode={mode}` in the subsection header (which seemed very unlikely because that's what this whole PR is about).

Maybe after "Push a new transaction." you could have a sentence or two noting that there's a variation of this endpoint that includes a `mode` parameter, documented below (with a link to it). Also note that this version (with no mode specified) is the same as using `mode=async`.

Reply:
I see you changed the order of these two versions of the POST /api/v1/transactions endpoint to have the one with the `mode` parameter first. That's more clear.